sdk/storage/azure-storage-blob/tests/_shared/avro/test_avro.py

```diff
@@ -0,0 +1,134 @@
+
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import inspect
+import os
+import unittest
+from io import BytesIO, open
+from azure.storage.blob._shared.avro.datafile import DataFileReader
+from azure.storage.blob._shared.avro.avro_io import DatumReader
+
+SCHEMAS_TO_VALIDATE = (
```

Comment:
Any specific reason to place the tests in the _shared folder? fyi, Just like the _shared folder in the code; we need to follow the same norms in the tests too. _shared folder will be common across packages.

Reply:
@rakshith91 It's testing `_shared` code so I was trying to match paths. But I see your point, so I'll move it.
However I wonder what would be a good place for such tests when `_shared` is actually automagically cloned across packages.

Reply:
I would put it an avro folder - looks like you already did - thanks :)